perf/test/angular-cli/tsconfig.json

```diff
@@ -3,11 +3,11 @@
   "compilerOptions": {
     "baseUrl": "./",
     "outDir": "./dist/out-tsc",
-    "noImplicitAny": true,
-    "noImplicitReturns": true,
-    "noImplicitThis": true,
+    "noImplicitAny": false,
```

Comment:
is it necessary to switch off these options?

Reply:
Otherwise I think it is good :)

Reply:
yeah, not really important since Stryker disables type checking. I tried to run the tests locally en then the compilation failed. The reason for that is probably because it is really old typescript code (stems from angular 4 days).